Dockerfile.dev

```diff
@@ -9,5 +9,9 @@ COPY . /opt/saltcorn
 RUN lerna bootstrap
 
 ENV PATH "$PATH:/opt/saltcorn/packages/saltcorn-cli/bin/saltcorn"
+ENV SALTCORN_MULTI_TENANT "true"
+ENV SQLITE_FILEPATH "/tmp/testdb.sqlite"
```

Comment:
Setting this environment variable will mean that we always use SQLite as the database